compiler/rustc_attr_parsing/src/attributes/cfg.rs

```diff
@@ -186,12 +188,13 @@ pub fn eval_config_entry(
     cfg_entry: &CfgEntry,
     id: NodeId,
     features: Option<&Features>,
+    emit_lints: bool,
```

Comment:
Since this boolean is used by "external" caller of attr parsing, could you make it explicit, maybe with a enum (so we can have documentation on the variants) about when it should be "true" or "false".